ui/src/alerting/actions/checks.ts

```diff
@@ -182,6 +182,18 @@ export const saveCheckFromTimeMachine = () => async (
   }
 }
 
+export const updateCheck = (check: Partial<Check>) => async (
```

Comment:
this was previously removed and caused a breaking change. It has been added back here with the intention of refactoring this implementation with a different ticket

Reply:
Yes, when I had originally submitted the comment I hadn't had the chance to write up the issue. Here's the breaking change PR:

https://github.com/influxdata/influxdb/pull/16333
but it seems like the changes made to updateCheck weren’t updated here:
https://github.com/influxdata/influxdb/blob/master/ui/src/alerting/components/CheckCard.tsx#L17

and here's the issue that is intended on resolving it :

https://github.com/influxdata/influxdb/issues/16339